cert/aziot-certd/src/http/mod.rs

```diff
@@ -3,36 +3,45 @@
 mod create;
 mod get_or_import_or_delete;
 
-pub(crate) struct Server {
-    pub(crate) inner: std::sync::Arc<futures_util::lock::Mutex<aziot_certd::Server>>,
+#[derive(Clone)]
+pub struct Service {
+    pub(crate) api: std::sync::Arc<futures_util::lock::Mutex<crate::Api>>,
 }
 
-http_common::make_server! {
-    server: Server,
+http_common::make_service! {
+    service: Service,
     api_version: aziot_cert_common_http::ApiVersion,
     routes: [
         create::Route,
         get_or_import_or_delete::Route,
     ],
 }
 
-fn to_http_error(err: &aziot_certd::Error) -> http_common::server::Error {
+fn to_http_error(err: &crate::Error) -> http_common::server::Error {
     let error_message = http_common::server::error_to_message(err);
 
     // TODO: When we get distributed tracing, associate these logs with the tracing ID.
     for line in error_message.split('\n') {
-        eprintln!("!!! {}", line);
+        log::log!(
+            if matches!(err, crate::Error::Internal(_)) {
```

Comment:
The intent is that `Internal` is the only one that maps to `Error` [1] and everything else maps to `Info`. It doesn't matter for certd and keyd because they only have two variants, but identityd has many more and so the if-else makes that clearer.

Or do you mean to merge it into the `match` below? If so, I left it as-is because it's better than repeating the loop in every variant's arm.

[1]: I just noticed I've written it backwards. I'll fix that.

Reply:
Oh, I meant it moreso as why are you using `matches!` if there's also an `else` branch

Right now, the code expands to:

```rust
    if match err {
        crate::Error::Internal(_) => true,
        _ => false,
    } {
        // ...
    } else {
        // ..
    }
```

Which just seems redundant

Reply:
Ah. I was proactvely correcting for https://rust-lang.github.io/rust-clippy/master/index.html#match_like_matches_macro ("one meaningful arm + one wildcard arm => use `matches!`"), but it only fires for bool exprs.

Done.